scripts/identify_changed_manifests_for_testing.py

```diff
@@ -1,69 +1,130 @@
+#!/usr/bin/env python
+""" 
+identify_changed_manifests_for_testing.py: Gather templates that have changed
+
+USAGE: 
+    python ./scripts/identify_changed_manifests_for_testing.py \
+    --changed_files_path ./tests/changed-files.txt \
+    --current_templates_url https://raw.githubusercontent.com/adknowledgeportal/data-models/main/modules/template/templates.csv \
+    --new_templates_path ./modules/template/templates.csv \
+    --csv_model_path ./AD.model.csv \
+    --template_config_path ./dca-template-config.json \
+    --output_file_path ./tests/changed-templates.json
+
+"""
+
+__author__ = "Abby Vander Linden"
+__status__ = "development"
 
 import os
-import pandas as pd
 import json
 import argparse
+import pandas as pd
+
+
+def get_templates_with_changed_attributes(
+    csv_model_path: str, changed_attributes: list
+) -> list:
+    """List of templates with changes in the data model
 
+    Args:
+        csv_model_path (str): relative path to csv data model
+        changed_attributes (list): _description_
 
-def get_templates_with_changed_attributes(csv_model_path: str, changed_attributes:list) -> list:
+    Returns:
+        list: name of templates with changes
+    """
     csv_model = pd.read_csv(csv_model_path)
-    changed_attributes_regex = '|'.join(changed_attributes)
-    templates_with_changed_attributes = csv_model[csv_model['DependsOn'].str.contains(changed_attributes_regex, na = False)]['Attribute'].to_list()
+    # changed_attributes_regex = '|'.join(changed_attributes)
+    templates_with_changed_attributes = csv_model[
+        csv_model["DependsOn"].isin(changed_attributes)
```

Comment:
this is exactly the kind of help I was looking for, things that don't need to be as many steps as I originally took 🙏🏻 . Once again reminded of the existince of `.isin`